### DISCUSSION TITLE
Is it possible to make a webbrowser with MSEgui ?

### DISCUSSION THREAD

**Discussion by beria23** [mse-org/mseide-msegui#51] [General]
Are there any msegui bindings for any browser that supports dynamic pages?
I searched everywhere because I already started my project with msegui and I don't want to go back to the less fast Lazarus....
ps: sorry, English, it's not my mother tongue...

**Comment by fredvs** [comment]
Hello.

Are you talking about a code-browser + form designer for MSEgui projects?
If so, there is MSEide: https://github.com/mse-org/mseide-msegui/releases
and ideU: https://github.com/fredvs/ideU/releases

Otherwise, please explain what is _browser that supports dynamic pages_
Thanks.

Fred

**Comment by beria23** [comment]
I was referring to a library or software package where I could get a page from the web, which is partially generated by the java script... Of course not only in MSWindows.
I don't even need to render it, because I'm not writing a browser, but it will be just a bonus...
In Lazarus this is done by geckoport,fpCEF3, CEF4Delphi, LazWebkit (Linux and Mac only) ....

**Comment by fredvs** [comment]
Hello.

Sorry but I dont have any skill in those Internet things.
There are lot of MSEgui internet components but I never try it.

I hope that somebody else can answer to your question.

**Comment by beria23** [comment]
Thank you very much anyway!!!!

**Comment by JonnyTech** [comment]
If you are happy to bind to an external library then https://github.com/PierceNg/fpwebview works very well.
